drivers/hmis/app/graphql/types/application/user.rb

```diff
@@ -30,6 +37,22 @@ class Application::User < Types::BaseObject
     field :date_created, GraphQL::Types::ISO8601DateTime, null: false
     field :date_deleted, GraphQL::Types::ISO8601DateTime, null: true
 
+    # audit_history returns the changes this user has made (as opposed to activity_logs which is just views, not edits).
+    # We use the generic term 'audit' to encompass both types of history (view and edit), but many places in the code,
+    # 'audit' just refers to edit history.
+    audit_history_field(
+      filter_args: { type_name: 'UserAuditEvent', omit: [:user] },
```

Comment:
ah I see the paper trail filter is not happy with this because `user` attribute is not defined on `input` anymore.

replace [this line](https://github.com/greenriver/hmis-warehouse/blob/c5f34ec587c96af1b99f7581242d3371f75fae1f/drivers/hmis/app/models/hmis/filter/paper_trail_version_filter.rb#L12) with the below fixes it, because the `with_filter` helper checks `respond_to?`

```
    with_filter(scope, :user) do
      scope = scope.where(v_t[:user_id].in(filters.user).or(v_t[:whodunnit].in(filters.user)).or(v_t[:true_user_id].in(filters.user)))
    end
```